### PR TITLE
Update Style + UX Writing for `MetamaskLogin` popup case `wrong-chain`

### PR DESCRIPTION
**Pull request by tsejerome**
https://github.com/amplify-labs/factor-app/pull/76
Updated style as below:
image

### CODE REVIEW

File: src/components/MetamaskLogin/template.pug [hunk 1]
```diff
@@ -20,17 +20,22 @@ mixin metamask-missing
...
     p.description 
-        | Please change your network to the {protocol} {' '}
+        | You are using a wrong network. Please connect to 
         b {networkName} {' '}
-        | network and refresh the page.
+        | to continue.
     .btns
+        Button(
+            label="How to?"
+            onClick="{init}"
```
Comment (georgeciubotaru) [inline src/components/MetamaskLogin/template.pug:41]: @tsejerome Please add a property for this onClick handler, this button should be a link

File: src/components/MetamaskLogin/template.pug [hunk 2]
```diff
@@ -20,17 +20,22 @@ mixin metamask-missing
         )
 
 mixin wrong-chain
-    .icon
-        img(loading="lazy" src="{metamaskImage}" alt="metamask icon")
+    .icon.popup__icon
```
Comment (georgeciubotaru) [inline src/components/MetamaskLogin/template.pug]: @tsejerome how about to put this class here `text-warning`?

File: src/components/MetamaskLogin/template.pug [hunk 3]
```diff
@@ -20,17 +20,22 @@ mixin metamask-missing
...
+        Icon(icon="{exclamation}" width="{60}" height="{60}" colorInherit)
```
Comment (georgeciubotaru) [inline src/components/MetamaskLogin/template.pug]: @tsejerome I think the width and height here is wrong, should be `52x52`

File: src/components/MetamaskLogin/template.pug (hunk 1 above)
Comment (tsejerome) [reply src/components/MetamaskLogin/template.pug:41]: What link shd it be, awaiting @markholdex or you for recommendation

File: src/components/MetamaskLogin/template.pug (hunk 2 above)
Comment (tsejerome) [reply src/components/MetamaskLogin/template.pug]: Text-warning will give it a red colour which was not in the design

File: src/components/MetamaskLogin/template.pug (hunk 1 above)
Comment (georgeciubotaru) [reply src/components/MetamaskLogin/template.pug:41]: @tsejerome  for now please put this URL
`https://metamask.zendesk.com/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC`

**Comment by tsejerome** [comment]
@georgeciubotaru shd be ready for merge

**Review by georgeciubotaru — APPROVED** [review]

File: src/components/MetamaskLogin/template.pug (hunk 2 above)
Comment (georgeciubotaru) [reply src/components/MetamaskLogin/template.pug]: @tsejerome please check the class values

**Pull request merged**